react/features/breakout-rooms/components/native/CollapsibleRoom.js

```diff
@@ -1,17 +1,17 @@
 // @flow
 
-import React, { useCallback, useState } from 'react';
+import React, { useCallback } from 'react';
 import { useTranslation } from 'react-i18next';
-import { FlatList, Text, TouchableOpacity, View } from 'react-native';
+import { FlatList } from 'react-native';
 import { useDispatch } from 'react-redux';
 
 import { openDialog } from '../../../base/dialog';
-import { Icon, IconArrowDown, IconArrowUp } from '../../../base/icons';
+import styles from '../../../participants-pane/components/native/styles';
```

Comment:
This looks very odd. Let's move this component to the participant-pane feature, since that's where it belongs.

Reply:
Will move to the participants-pane feature.